server/app.js

```diff
@@ -1,3 +1,4 @@
+require("dotenv").config();
```

Comment:
conditionally render this if it's in development and not production. Once we deploy to Heroku, this will crash the app as it's a dev dependency, and Heroku doesn't install dev dependencies.